modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -18,16 +18,11 @@ The 3.x Python SDK introduces comprehensive https://www.python.org/dev/peps/pep-
 
 == Requirements
 
-
 Couchbase Python SDK bundles libcouchbase automatically, so no need to install it separately.
 You may need CMake to install, although the installer will attempt to download it from PyPI automatically.
 
 The Python SDK 3.x requires Python 3, with Python 3.5 and above supported.
 
-// end::prep[]
```

Comment:
Did you mean to get rid of the tags?

Reply:
Yes and no -- I didn't know what they were in reference to and they looked commented out.  Are they used for anything?  If not, I will update to remove the // tag::prep[] and // end::install[] that is still in the doc.